backend/prompt_studio/prompt_studio_core/document_indexing_service.py

```diff
@@ -8,32 +8,45 @@ class DocumentIndexingService:
     CACHE_PREFIX = "document_indexing:"
 
     @classmethod
-    def set_document_indexing(cls, doc_id_key: str) -> None:
+    def set_document_indexing(cls, org_id: str, user_id: str, doc_id_key: str) -> None:
```

Comment:
@muhammad-ali-e can we get the org_id and user_id from the ContextVar of the request being processed? We set the `log_events_id` through [this StateStore](https://github.com/Zipstack/unstract/blob/9c1948be354221c822834d20aa346d0f28b5f014/backend/account/custom_auth_middleware.py#L38-L41)
NIT: @jagadeeswaran-zipstack if this can be used safely we can make use of this as well

Reply:
We aleready have these functions in UserSessionUtils (backend/utils/user_session.py)

Reply:
@muhammad-ali-e those functions seem to accept a request for this. In this case also we'd have to pass down the request everywhere to get these details. If its safe, we could add this org_id and user_id to the StateStore and fetch it from there
CC: @hari-kuriakose